linera-base/src/crypto/secp256k1.rs

```diff
@@ -63,7 +63,7 @@ pub struct Secp256k1Signature(pub Signature);
 
 impl Secp256k1PublicKey {
     /// A fake public key used for testing.
-    #[cfg(with_testing)]
+    #[cfg(all(with_testing, not(target_arch = "wasm32")))]
```

Comment:
Did you encounter problems with compilation here?

Reply:
Yes, I believe when it tried to compile the tests for `wasm32-unknown-unknown`.